src/main/java/net/kodehawa/mantarobot/commands/custom/v3/serializer/SerializerVisitor.java

```diff
@@ -26,6 +26,8 @@
 import org.json.JSONArray;
 import org.json.JSONObject;
 
+// this class might legitimately be unused by v3 is above my head
+@SuppressWarnings("unused")
 public class SerializerVisitor implements NodeVisitor<JSONObject, Void> {
```

Comment:
@natanbc this is the one Kode meant to actually ask you about whether or not we can remove it.